nebullvm/api/functions.py

```diff
@@ -108,6 +136,44 @@ def _is_huggingface_data(data_sample: Any) -> bool:
     return False
 
 
+def _benchmark_original_model(
+    model: Any,
+    input_data: DataManager,
+    dl_framework: DeepLearningFramework,
+    compute_output: bool = False,
+):
+    device = "cuda" if torch.cuda.is_available() else "cpu"
+    outputs = None
+
+    input_data = input_data.get_numpy_list(QUANTIZATION_DATA_NUM)
+    input_data = [
+        tuple(convert_to_target_framework(t, dl_framework) for t in data_tuple)
+        for data_tuple in input_data
+    ]
+    if compute_output:
+        outputs = [
+            tuple(
+                COMPUTE_OUTPUT_FRAMEWORK[dl_framework](
+                    model, list(input_tensors)
+                )
+            )
+            for input_tensors in input_data
+        ]
+    latency, _ = COMPUTE_LATENCY_FRAMEWORK[dl_framework](
```

Comment:
in this way for the original model we are computing the latency just for the first data sample! We should compute it for the whole `input_data` and then average the results. Note that the functions `compute_*_latency` by default repeat the experiment 100 times. If we start computing the latency for 100 samples it is un-necessary to continue repeating 100 times each experiment too.